packages/bpk-tokens/tokens/base.raw.ios.json

```diff
@@ -87,9 +87,6 @@
     "FONT_WEIGHT_BOLD": {
       "value": "700"
     },
-    "FONT_WEIGHT_HEAVY": {
```

Comment:
This was unused - aliases are not included in output (apart from raw but these token formats are not "advertised") so non breaking change.